platforms/enterprise/enterprise/src/main/java/org/gradle/internal/enterprise/impl/legacy/DevelocityPluginCompatibility.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2025 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.internal.enterprise.impl.legacy;
+
+import com.google.common.annotations.VisibleForTesting;
+import org.gradle.util.internal.VersionNumber;
+
+/**
+ * Information about the earliest supported Develocity plugin version.
+ */
+public class DevelocityPluginCompatibility {
+
+    // Gradle versions 9+ are not compatible with Gradle Enterprise plugin < 3.13.1
+    @VisibleForTesting
+    public static final String MINIMUM_SUPPORTED_PLUGIN_VERSION = "3.13.1";
+    @VisibleForTesting
+    public static final VersionNumber MINIMUM_SUPPORTED_PLUGIN_VERSION_NUMBER = VersionNumber.parse(MINIMUM_SUPPORTED_PLUGIN_VERSION);
+
+    @VisibleForTesting
+    public static final String ISOLATED_PROJECTS_SUPPORTED_PLUGIN_VERSION = "3.15";
+    @VisibleForTesting
+    public static final VersionNumber ISOLATED_PROJECTS_SUPPORTED_PLUGIN_VERSION_NUMBER = VersionNumber.parse(ISOLATED_PROJECTS_SUPPORTED_PLUGIN_VERSION);
```

Comment:
🤔 It might make sense to hard-code this version in tests instead of re-using it, so we at least see what things change/break when we change the minimum supported plugin version. I don't feel very strongly though.